src/x11/window.rs

```diff
@@ -263,7 +271,9 @@ impl Window {
     // FIXME: poll() acts fine on linux, sometimes funky on *BSD. XCB upstream uses a define to
     // switch between poll() and select() (the latter of which is fine on *BSD), and we should do
     // the same.
-    fn run_event_loop<H: WindowHandler>(&mut self, handler: &mut H) {
+    fn run_event_loop<H: WindowHandler>(&mut self, handler: &mut H, message_rx: Consumer<H::Message>) {
+        let mut message_rx = message_rx;
```

Comment:
This is a minor thing, but you can have mut in the function argument instead of here, like _mut message_rx: Consumer<H::Message>_.

Reply:
Oh, ha! I did not know that.